src/scss/_global/_structure.scss

```diff
@@ -27,10 +83,17 @@ hr {
   border: 0;
   color: $hr-color;
   height: 1px;
-  margin: 2.75rem 0;
+  margin: 2.5rem 0;
   position: relative;
 }
 
 .divider {
-  width: 100%;
+  left: -1000%;
+  position: relative;
+  width: 4000%;
+}
+
+.horizontal-space {
+  position: sticky;
```

Comment:
The idea is to allow us to build layouts with content set at varying widths, like in this screenshot

![2016_07_14_12_10_13_tax_foundation_style_guide](https://cloud.githubusercontent.com/assets/10962539/16846895/41b6685c-49bc-11e6-9863-37ce47aab117.png)